src/main/java/frc/robot/RobotContainer.java

```diff
@@ -177,11 +167,17 @@ private void configureBindings() {
       
     }, swerveDrive));
 
+    shooter.setDefaultCommand(new RunCommand(() -> {
+      shooter.feed(0);
+    }, shooter));
+
+    intakeSubsystem.setDefaultCommand(new RunCommand(() -> {
+      intakeSubsystem.intake(0);
+    }, intakeSubsystem));
+
```

Comment:
This is fine. Alternatively, in the constructor for these subsystems you can do

```
setDefaultCommand(run(() -> feed(0)));

setDefaultCommand(run(() -> intake(0)));
```

Reply:
Likewise, all of the `new RunCommands` _could be_ converted to use the command factory methods. Not necessary, just a little simpler code

Reply:
I like leaving these default commands in the robot container class so that they are next to all of the inputted commands, I think it makes the most sense even though that code is definitely cleaner.